src/karts/skidding.cpp

```diff
@@ -443,6 +443,9 @@ void Skidding::update(float dt, bool is_on_ground,
                                              bonus_speed, bonus_speed,
                                              bonus_force, bonus_time,
                                              /*fade-out-time*/ 1.0f);
+                    // Play custom character sound (weee!)
+                    m_kart->playCustomSFX(SFXManager::CUSTOM_SKID);
+                    m_kart->m_controller->handleZipper(true);                   
```

Comment:
The goal is to play the sound + have the blur effect. If i get it right i must do a copy of 
void LocalPlayerController::handleZipper(bool play_sound)  in src/karts/controller/local_player_controller.cpp
void PlayerController::handleZipper(bool play_sound) in src/karts/controller/player_controller.cpp
and customise them ? in order to have a more independent effect